Source/WebCore/css/CSSFontFaceSrcValue.h

```diff
@@ -59,9 +60,56 @@ class CSSFontFaceSrcLocalValue final : public CSSValue {
     WeakPtr<SVGFontFaceElement, WeakPtrImplWithEventTargetData> m_element;
 };
 
+enum class FontTech : uint8_t {
+    ColorColrv0,
+    ColorColrv1,
+    ColorCbdt,
+    ColorSbix,
+    ColorSvg,
+    FeaturesAat,
+    FeaturesGraphite,
+    FeaturesOpentype,
+    Incremental,
+    Palettes,
+    Variations,
+    // Reserved for invalid conversion result.
+    Invalid
+};
+
+inline String cssTextFromFontTech(FontTech tech)
```

Comment:
Should return an ASCIILiteral. We don't need an unnecessary String allocation here.